packages/world/src/utils/InventoryUtils.sol

```diff
@@ -135,6 +135,7 @@ function useEquipped(
     } else {
       // use 10% of the mass if it's greater than 10
       if (toolMassLeft > 10) {
+        // TODO: shouldn't it be a percentage of the max mass?
```

Comment:
but you might not be able to use 10% of the max if you have less than that right?

Reply:
In that case you would use all the remaining mass maybe. Idk, it didn't make sense to me conceptually that every time you mine it uses less mass

Reply:
yea agree what you're suggesting makes more sense. lets change it to that! could you create an issue for it?